### DISCUSSION TITLE
he POST method is not supported for route

### DISCUSSION THREAD

**Discussion by ValerianMemsk** [orchidsoftware/platform#2682] [Q&A]
Using  v13.6
Created screen with 2 different modals 

```
ModalToggle::make('app.reports.title')
                    ->modal('mailReports')
                    ->method('setUpMailReports')
                    ->icon('envelope'),

ModalToggle::make('app.uploadFile')
                    ->modal('fileUpload')
                    ->method('handleUpload')
                    ->icon('cloud-upload'),
```
Booth are simpe - few inputs and button
Both using same "skeleton": `function ... (Request $request) {}`

Second one works ok, first one causes error `The POST method is not supported for route admin/leftovers/setUpMailReports. Supported methods: GET, HEAD`

How could this even be happening? Looks absurd.
And yes, i read similar discussion here - its not helpful at all. Yeah, i get it - routes interpreted incorrectly. There is not a word in documentation how to fix it, or at least i didn't find any.

**Comment by darrylk** [comment]
This is a pretty old issue but I just ran into this myself and it turned out for me that I had the route itself defined as Route::get( . . .and I just had to change it to Route::post( . . . to get it to work.
